### DISCUSSION TITLE
(question) Pro version

### DISCUSSION THREAD

**Discussion by dragonGod97677** [d4rken-org/sdmaid-se#532] [Q&A] [answered]
There will be a donation app (to unlock pro functions) on sd maid 2? Like on the original sd maid.

**Comment by d4rken** [comment, marked as answer]
Nope. Also not that great a solution currently due to not being able to hide the unlocker anymore.

The unlocker only exists because when SD Maid 1 launched, in-app purchases didn't exist yet.

**Reply by babeuloula** [reply]
Hello,

So we need to rebuy the app to have all the features ?

Regards

**Reply by dragonGod97677** [reply]
Yes but we will supporting him 😃
